src/main/java/uk/gov/hmcts/reform/sscs/service/RegionalProcessingCenterService.java

```diff
@@ -130,4 +131,11 @@ public static String getFirstHalfOfPostcode(String postcode) {
         return "";
     }
 
+    public HearingRoute getHearingRoute(String region) {
+        return regionalProcessingCenterMap.values().stream()
+            .filter(rpc -> rpc.getName().equalsIgnoreCase(region))
+            .map(RegionalProcessingCenter::getHearingRoute)
+            .findFirst().orElse(HearingRoute.LIST_ASSIST);
```

Comment:
You have an uncovered test case for the default

Reply:
Added